students/steve-long/lesson02-basic-functions/ex-2-2-fizzBuzz/fizzBuzz.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# ========================================================================================
+# Python210 | Fall 2020
+# ----------------------------------------------------------------------------------------
+# Lesson02
+# Fizz Buzz Exercise (fizzBuzz.py)
+# Steve Long 2020-09-19 | v0
+#
+# Requirements:
+# -------------
+# Write a program that prints the numbers from 1 to 100 inclusive.
+# But for multiples of three print "Fizz" instead of the number.
+# For the multiples of five print "Buzz" instead of the number.
+# For numbers which are multiples of both three and five print "FizzBuzz" instead.
+#
+# Implementation:
+# ---------------
+#	fizzBuzz([<start>[,<end>]])
+#
+# Script Usage:
+# -------------
+# 	python fizzBuzz.py [<start> [<end>]]
+#		<start> ::= Starting number (see function fizzBuzz for details.)
+#		<end>   ::= Ending number (see function fizzBuzz for details.)
+#
+# History:
+# --------
+# 000/2020-09-20/sal/Created.
+# ========================================================================================
+
+import sys
+	
+def fizzBuzz(start = 1, end = 100):
```

Comment:
per python best practices, function names and variable names follow snake_case convention -> `def fizz_buzz`